product_cost_security/models/product.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Sergio Teruel - Tecnativa <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    standard_price = fields.Float(
+        groups='product_cost_security.group_product_cost',
```

Comment:
Maybe you should add the group the affected views, as otherwise breaks them: 
![image](https://user-images.githubusercontent.com/5040182/48632523-64913d00-e9c1-11e8-8fbd-f8e62850b548.png)

Reply:
I'd say this is an Odoo's bug, anyway: https://github.com/odoo/odoo/blob/11.0/addons/web/static/src/js/views/form/form_renderer.js#L650

PR here: https://github.com/odoo/odoo/pull/28770